e2e/bitcoin-spv.sh

```diff
@@ -22,20 +22,18 @@ if ! docker exec -it bitcoind-node bitcoin-cli -regtest getblockchaininfo; then
 fi
 
 # query the lightclient for the latest block
-chaintip_response_before=$(curl --user user:password --data-binary '{"jsonrpc":"1.0","id":"curltest","method":"get_header_chain_tip","params":[]}' -H 'content-type: text/plain;' http://127.0.0.1:9797)
+chaintip_response_before=$(sui client call --function latest_block_hash --module light_client --package $PACKAGE_ID --gas-budget 10000000 --args $OBJECT_ID --dev-inspect --json)
```

Comment:
Where you define OBJECT_ID and PACKAGE_ID?

Reply:
In the environment

Reply:
Can add a README for it

Reply:
Yes, please go ahead.